modules/@apostrophecms/piece-type/ui/apos/components/AposImageCropper.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="apos-image-cropper">
+    <cropper
+      ref="cropper"
+      :src="attachment._urls.uncropped ? attachment._urls.uncropped.original : attachment._urls.original"
```

Comment:
If we do https://github.com/apostrophecms/apostrophe/pull/3708/files#r853039376:

```suggestion
      :src="attachment._urlsUncropped ? attachment._urlsUncropped.original : attachment._urls.original"
```